drivers/adc/adc_nrfx_saadc.c

```diff
@@ -75,8 +75,6 @@ static const uint32_t saadc_psels[NRF_SAADC_DVDD + 1] = {
 	[NRF_SAADC_AIN5] = NRF_PIN_PORT_TO_PIN_NUMBER(10U, 1),
 	[NRF_SAADC_AIN6] = NRF_PIN_PORT_TO_PIN_NUMBER(11U, 1),
 	[NRF_SAADC_AIN7] = NRF_PIN_PORT_TO_PIN_NUMBER(12U, 1),
-	[NRF_SAADC_VDD]  = NRF_SAADC_INPUT_VDD,
-	[NRF_SAADC_DVDD] = NRF_SAADC_INPUT_DVDD,
```

Comment:
The array size needs to be updated accordingly.

Reply:
fixed